app-example/src/main/java/com/badoo/ribs/example/app_bar/AppBar.kt

```diff
@@ -0,0 +1,25 @@
+package com.badoo.ribs.example.app_bar
+
+import com.badoo.ribs.core.Rib
+import com.badoo.ribs.core.customisation.RibCustomisation
+import com.badoo.ribs.example.image.ImageDownloader
+import com.badoo.ribs.example.repository.UserRepository
+import io.reactivex.functions.Consumer
+
+interface AppBar : Rib {
+
+    interface Dependency {
+        val userRepository: UserRepository
+        val appBarOutput: Consumer<Output>
```

Comment:
I think it would be better to use Connectable for the input/output of rib. @zsoltk what do you think?

Reply:
Does it make sense to have a Connectable when there is no input? Something like Connectable<Nothing, Output>?

Reply:
+1

Reply:
Sorry intended to write +1 as a reply to first comment.

Instead of Nothing you can have an empty `sealed class Input`, seems to be easier as a pattern to use.

Reply:
Yes we can use nothing, also we can just leave input empty. It shouldn't add much overhead

Reply:
Alright will use empty Input